client/src/components/AddClimbedBoulderForm.jsx

```diff
@@ -0,0 +1,184 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const AddClimbedBoulderForm = ({saveClimbedBoulder}) => {
+  const [projected, setProjected] = useState(false);
+  const [attempts, setAttempts] = useState("");
+  const [result, setResult] = useState();
+  const [liked, setLiked] = useState(false);
+  const [levelFeedback, setLevelFeedback] = useState();
+  
+
+  return (
+    <BoulderForm
+      onSubmit={(event) => {event.preventDefault();
+        saveClimbedBoulder(projected, attempts, result, liked, levelFeedback);
+      }}
+    >
+      <p>Save your achievement!</p>
```

Comment:
I think it would be more semantic to use a header here, what do you think?

Reply:
Yes, have changed it.